testplan/testing/multitest/driver/fix/client.py

```diff
@@ -215,7 +215,7 @@ def logoff(self):
         self._client.sendlogoff()
         rcv = self._client.receive(timeout=self.cfg.logoff_timeout)
         self.file_logger.debug('Received logoff response {}.'.format(rcv))
-        if 35 not in rcv or rcv[35] != b'5':
+        if 35 not in rcv or rcv[35] not in (b'5', u'5'):
```

Comment:
same comment here regarding message values being either bytes or unicode, it's not good. Better to ensure they are always either bytes or unicode as makes the most sense (do the values represent text or non-text data?)

Reply:
we init the fixServer(name='server',  msgclass=FixMessage, codec=CODEC)
Here we use FixMessage from pyfixmsg library which has been updated to be compatible with Py3, so we can directly write **rcv[35] != '5'** here. Although use can pass their customized class as argument and then we can get b'5', u'5' or anything else, but maybe it's their responsibility to let their parser work properly and write py2/3 compatible code, if they really want to use customized class instead of default one. So I just check if value of tag 35 is '5', also changed some similar code in this PR.